bundles/org.openhab.binding.lgthinq/README.md

```diff
@@ -0,0 +1,195 @@
+| Express Freeze                | Switch      | Channel to change the express freeze function (ON/OFF/Rapid)                   |
+| Express Cool                  | Switch      | Channel to switch ON/OFF express cool function                                 |
+| Vacation                      | Switch      | Channel to switch ON/OFF Vacation function (unit will work in eco mode)        | 
+
+#### More Information
+
+This Channel Group is reports useful information data for the device:
+
+| channel          | type      | description                                                |
+|------------------|-----------|------------------------------------------------------------|
+| Fresh Air Filter | String    | Shows the Fresh Air filter status (OFF/AUTO/POWER/REPLACE) |
+| Water Filter     | String    | Shows the filter's used months                             |
+
+OBS: some versions of this device can not support all the channels, depending on the model's capabilities.
+
+**Important:** this binding will always interact with the LG API server to get information about the device. This is the Smart ThinQ way to work, there is no other way (like direct access) to the devices. Hence, some side effects will happen in the following situations:
+1. **Internet Link** - if you OpenHab server doesn't have a good internet connection this binding will not work properly! In the same way, if the internet link goes down, your Things and Bridge going to be Offline as well, and you won't be able to control the devices though OpenHab until the link comes back.
+2. **LG ThinQ App** - if you've already used the LG ThinQ App to control your devices and hold it constantly activated in your mobile phone, you may experience some instability because the App (and Binding) will try to lock the device in LG ThinQ API Server to get it's current state. In the app, you may see some information in the device informing that "The device is being used by other" (something like this) and in the OpenHab, the thing can go Offline for a while.
+3. **Pooling time** - both Bridge and Thing use pooling strategy to get the current state information about the registered devices. Note that the Thing pooling time is internal and can't be changed (please, don't change in the source code) and the Bridge can be changed for something greater than 300 seconds, and it's recommended long pooling periods for the Bridge because the discovery process fetch a lot of information from the LG API Server, depending on the number of devices you have registered in your account. 
+About this last point, it's important to know that LG API is not Open & Public, i.e, only LG Official Partners with some agreement have access to their support and documentations. This binding was a hard (very hard actually) work to dig and reverse engineering in the LG's ThinQ API protocol. Because this, you must respect the hardcoded pool period to do not put your account in LG Blacklist.
```

Comment:
```suggestion
**Important:** this binding will always interact with the LG API server to get information about the device. 
This is the Smart ThinQ way to work, there is no other way (like direct access) to the devices.
Hence, some side effects will happen in the following situations:
1. **Internet Link** - if you openHAB server doesn't have a good internet connection this binding will not work properly! 
In the same way, if the internet link goes down, your Things and Bridge going to be Offline as well, and you won't be able to control the devices though openHAB until the link comes back.
2. **LG ThinQ App** - if you've already used the LG ThinQ App to control your devices and hold it constantly activated in your mobile phone, you may experience some instability because the App (and Binding) will try to lock the device in LG ThinQ API Server to get it's current state.
In the app, you may see some information in the device informing that "The device is being used by other" (something like this) and in the openHAB, the thing can go Offline for a while.
3. **Polling time** - both Bridge and Thing use a polling strategy to get the current state information about the registered devices.
Note that the Thing polling time is internal and can't be changed (please, don't change in the source code) and the Bridge can be changed for something greater than 300 seconds, and it's recommended long pooling periods for the Bridge because the discovery process fetch a lot of information from the LG API Server, depending on the number of devices you have registered in your account. 
About this last point, it's important to know that LG API is not Open & Public, i.e, only LG Official Partners with some agreement have access to their support and documentations.
This binding was a hard (very hard actually) work to dig and reverse engineering in the LG's ThinQ API protocol.
Because this, you must respect the hardcoded pool period to do not put your account in LG Blacklist.
```

Each sentence shoudl go on its own line. Besides that two comments: I wonder how usefull it is to explain the cloud api dependency. As this is the same case for most addons. It also states taht the polling interval is hardcoded, but it is also part of the thing configuration if im not mistaken.

Reply:
I agree with you. I will remove the Cloud API explanations to have more concise documentation.